src/main/java/org/mule/extension/ftp/internal/FtpUtils.java

```diff
@@ -26,4 +27,11 @@ public static String normalizePath(String path) {
     return FilenameUtils.normalize(path, true);
   }
 
+  public static String getReplyCodeErrorMessage(Integer replyCode) {
+    if (FTPReply.isPositiveCompletion(replyCode)) {
```

Comment:
I would use a ternary operator here. 

return FTPReply.isPositiveCompletion(replyCode) ? "" :  "The error message";